google/cloud/osconfig/README.md

```diff
@@ -1,16 +1,12 @@
 # OS Config API C++ Client Library
 
-:construction:
-
 This directory contains an idiomatic C++ client library for the
 [OS Config API][cloud-service-docs], a service for patch management, patch
 compliance, and configuration management on VM instances.
 
 <!-- TODO(#8139) - fix streaming read updated namespace before GA -->
```

Comment:
Can you remove the TODO entry too?  It seems it no longer applies.

Reply:
Done.